integration/src/integration/java/pl/allegro/tech/hermes/integration/management/GroupManagementTest.java

```diff
@@ -16,6 +18,71 @@
 
 public class GroupManagementTest extends IntegrationTest {
 
+    @Test
+    public void shouldEmmitAuditEventWhenGroupCreated() {
+        //given
+        RemoteServiceEndpoint remoteService = new RemoteServiceEndpoint(SharedServices.services().serviceMock(), "/audit-events");
+        String expectedBody = "{\n" +
+                "  \"eventType\": \"CREATED\",\n" +
+                "  \"resourceName\": \"Group\",\n" +
+                "  \"payloadClass\": \"pl.allegro.tech.hermes.api.Group\",\n" +
+                "  \"payload\": {\n" +
+                "    \"groupName\": \"exampleGroup\",\n" +
+                "  },\n" +
+                "  \"username\": \"[anonymous user]\"\n" +
+                "}";
+
+        //when
+        management.group().create(group("exampleGroup").build());
+
+        //then
+        assertThat(remoteService.waitAndGetLastRequest().getBodyAsString().equals(expectedBody));
```

Comment:
Does this actually assert anything? According to [AssertJ's docs](https://assertj.github.io/doc/#forgetting-to-call-an-assertion) the assertion is not called in such cases. What you are essentially doing here is calling `assertThat()` with a boolean value. What you should do is call
```java
assertThat(
  remoteService.waitAndGetLastRequest().getBodyAsString()
).isEqualTo(expectedBody)
```

Please review other instances of using AssertJ, too.